### PR TITLE
first test of new exercise folder

### PR DESCRIPTION
**Pull request by Entkenntnis**
preview: https://frontend-git-setup-new-view-for-exercise-folders-serlo.vercel.app/66809

ready for release

### CODE REVIEW

**Comment by Entkenntnis** [comment]
- [x] increase shadow lg
- [x] call to action zurück zum menu
- [x] Aufgabe mit Teilaufgaben: sub screens
- [x] zentrieren beim kleinsten breaking point

next step: Sortierung/Filterfunktion z.B. nach Schwierigkeit nächste Iteration

**Comment by Entkenntnis** [comment]
task today: technical prototype for sub-exercises

**Comment by Entkenntnis** [comment]
remaining:

- [x] change exercise solved state to highlight buttons in sub exercises

File: src/components/taxonomy/new-folder-prototype.tsx [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+      })}
+    </div>
+  )
+}
+
+function renderDifficulty(dif: number) {
+  if (dif === 1) {
+    return (
+      <>
+        <span className="inline-block w-14">leicht</span> ★☆☆
```
Comment (elbotho) [inline src/components/taxonomy/new-folder-prototype.tsx:216]: I'd prefer ●○○ and so on since stars suggest quality of the exercise to me.

**Review by elbotho — COMMENTED** [review]
Commented one small thing.
@Entkenntnis what kind of feedback would you like?
If this was production code I'd would have some remarks but since this is a prototype… :)

File: src/components/taxonomy/new-folder-prototype.tsx (hunk 1 above)
Comment (Entkenntnis) [reply src/components/taxonomy/new-folder-prototype.tsx:216]: thanks for the feedback, we will take a look at it while doing usability tests

**Pull request merged**